raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Control of litter in public places 
+(121) [1] Every person who has a public place under that person's control or management shall at all times provide and maintain in that place, where litter is likely to be deposited, such number of litter receptacles of suitable construction and design for the temporary deposit of litter as may reasonably be necessary to keep that place free from litter. [2] Where litter generated on or attributable to any particular premises is likely to be carried from or to otherwise escape from those premises onto a public place, an Environment Inspector may require the occupier of the premises to take all reasonable steps to prevent such litter being carried to or escaping onto the public place. [3] Where it is shown that, excessive litter is attributable to or emanates from any particular premises, an Environment Inspector may require the occupier of the premises to provide and maintain such number of litter receptacles of suitable construction or design in any public place adjacent to or within the vicinity of the land or premises for the temporary deposit of litter, as may reasonably be necessary to ensure that the public place be kept free of that litter. [4] Where any occupier fails to comply within a reasonable time with a requirement of an Environment Inspector made under subsection [2] or [3], the officer may take any reasonable steps to remedy that default and shall recover the cost of so doing from the occupier as a debt due to the service. [5] Every person to whom this section applies shall also make appropriate provision for emptying the contents of litter receptacles provided within the public places under that person's control or management, and for the removal and disposal of those contents promptly, efficiently and at regular intervals. 
```

Comment:
Please replace `Where it is shown that, excessive` with `Where it is shown that excessive`.